Examples/MAX32520/Hello_World_Cpp/main.cpp

```diff
@@ -1,35 +1,3 @@
-/******************************************************************************
- * Copyright (C) 2023 Maxim Integrated Products, Inc., All Rights Reserved.
```

Comment:
Script fails for .cpp files

Reply:
Thanks. I added this file type to the copyright script.